src/Options/Harg/Single.hs

```diff
@@ -59,4 +59,4 @@ instance B.TraversableB (Single a) where
 
 instance B.ApplicativeB (Single a) where
   bprod (Single l) (Single r) = Single (P.Pair l r)
-  bpure = Single
+  bpure x = Single x
```

Comment:
I believe this is required due to the new [simplified subsumption](https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/0287-simplify-subsumption.rst) (in GHC > 9.0)